lib/logger.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+var prettyMs = require('pretty-ms');
+var figures = require('figures');
+var Squeak = require('squeak');
+var chalk = require('chalk');
+var plur = require('plur');
+var log = new Squeak({separator: ' '});
+var x = module.exports;
+
+function beautifyStack(stack) {
+	var re = /(?:^(?! {4}at\b).{6})|(?:\((?:[\\\/](?:(?!node_modules[\\\/]ava[\\\/])[^:\\\/])+)+:\d+:\d+\))/;
+	var found = false;
+
+	return stack.split('\n').filter(function (line) {
+		var relevant = re.test(line);
+		found = found || relevant;
+		return !found || relevant;
+	}).join('\n');
+}
+
+log.type('success', {
+	color: 'green',
+	prefix: figures.tick
+});
+
+log.type('error', {
+	color: 'red',
+	prefix: figures.cross
+});
+
+x.write = log.write.bind(log);
+x.writelpad = log.writelpad.bind(log);
+x.success = log.success.bind(log);
+x.error = log.error.bind(log);
+
+x.test = function (err, title, duration) {
+	var isForked = process.env.AVA_FORK;
+
+	if (isForked) {
+		var data = JSON.stringify({
+			err: {
+				message: (err || {}).message,
+				stack: (err || {}).stack
+			},
+			title: title,
+			duration: duration
+		});
+
+		var stream = err ? process.stderr : process.stdout;
+		stream.write(data);
+		return;
+	}
+
+	if (err) {
+		log.error(title, chalk.red(err.message));
+		return;
+	}
+
+	// if (runner.stats.testCount === 1) {
+	// 	return;
+	// }
```

Comment:
For `logger` to access a `runner` object, there would need to be some kind of initialization to get a reference to it. For example:

```js
var Runner = require('./lib/runner');
var log = require('./lib/logger');

var runner = new Runner();

log.init(runner);
```

So I decided to skip it for now.